client/state/current-user/gravatar-status/test/selectors.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { set } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	isCurrentUserUploadingGravatar
+} from '../selectors';
+
+describe( 'selectors', () => {
+	describe( '#isCurrentUserUploadingGravatar', () => {
+		it( 'returns false when state is undefined', () => {
+			expect( isCurrentUserUploadingGravatar( undefined ) ).to.equal( false );
+		} );
+
+		it( 'returns state when defined', () => {
+			const uploadingState = {};
+			set( uploadingState, 'currentUser.gravatarStatus.isUploading', true );
+			expect( isCurrentUserUploadingGravatar( uploadingState ) )
```

Comment:
I don't have a huge preference, using `set` just seemed neater to me as I was writing it out. There isn't a non-mutating lodash method for this. I updated it to use object literals. Had to make it multiline, but that's ok :)